### PR TITLE
BGP Neighbors module unit testing code added

### PR DESCRIPTION
**Pull request by kannankvs**
##### SUMMARY
BGP Neighbors module unit testing code added

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
BGP Neighbors Unit Testing

### CODE REVIEW

File: tests/unit/modules/network/sonic/fixtures/sonic_bgp_neighbors.yaml
Comment (kerry-meyer) [inline tests/unit/modules/network/sonic/fixtures/sonic_bgp_neighbors.yaml:1]: The UT coverage on the "config" file for this resource module is only 59%. My understanding is that this is below the target. If there are additional attributes that can be added to the test suite to increase the coverage, please add them. If there is a reason why the target can not be met at this time, please explain the considerations.

```
plugins/module_utils/network/sonic/config/bgp_neighbors/bgp_neighbors.py                  961    327    700    175   59%
```

**Review by kerry-meyer — CHANGES_REQUESTED** [review]
I have posted a comment in the ".yaml" file: Please see if it is possible to increase the coverage above the current 59% level.

File: tests/unit/modules/network/sonic/fixtures/sonic_bgp_neighbors.yaml
Comment (kannankvs) [reply tests/unit/modules/network/sonic/fixtures/sonic_bgp_neighbors.yaml:1]: @kerry-meyer : UT coverage improvement is done. Kindly review.

Comment (kerry-meyer) [reply tests/unit/modules/network/sonic/fixtures/sonic_bgp_neighbors.yaml:1]: Thanks for adding the additional coverage (and all of the other UT).

Approved.

**Review by kerry-meyer — APPROVED** [review]
Thanks for providing the UT for this module and making the additional changes to increase coverage.

Approved.

**Pull request merged**